test/unittests/UtilityTest.php

```diff
@@ -46,13 +46,10 @@ class UtilityTest extends TestCase
      * @var array contains project information retrieved by getProjectList method
      */
     private $_projectInfo = array(
-        array('ProjectID' => '12',
-              'Name' => 'project1',
-              'recruitmentTarget' => '123456'),
-        array('ProjectID' => '23',
-              'Name' => 'project2',
-              'recruitmentTarget' => '234567')
-        );
+        '12' => 'project1',
```

Comment:
changes were made to resolve Failures

https://travis-ci.org/aces/Loris/jobs/572998062